src/battle_util.c

```diff
@@ -8379,7 +8377,7 @@ u8 IsMonDisobedient(void)
     }
 
     if (B_OBEDIENCE_MECHANICS >= GEN_8
-     && !IsOtherTrainer(gBattleMons[gBattlerAttacker].otId, gBattleMons[gBattlerAttacker].otName))
+        && !IsOtherTrainer(gBattleMons[gBattlerAttacker].otId, gBattleMons[gBattlerAttacker].otName))
```

Comment:
We are trying to use the the first formatting instead of what it was just change into. At least for new stuff.